src/libAtomVM/term.h

```diff
@@ -705,6 +705,27 @@ static inline term term_from_literal_binary(const void *data, uint32_t size, Con
     return ((term) boxed_value) | TERM_BOXED_VALUE_TAG;
 }
 
+/**
+* @brief Create an empty binary.  All bytes in the binary are initialized to 0x00
+*
+* @details Allocates a binary on the heap, and returns a term pointing to it.
+* @param size size of binary data buffer.
+* @param ctx the context that owns the memory that will be allocated.
+* @return a term pointing to the boxed binary pointer.
+*/
+static inline term term_create_empty_binary(uint32_t size, Context *ctx)
```

Comment:
I would rather implement term_create_uninitialized_binary, and avoid the next memset.
Such a similar function would be useful in a lot of different scenarios.

Reply:
Yeah, I lifted it from the bit syntax PR :)  In that PR, I kind of need the memset, but there is nothing that says I can't do the memset outside of this function.  I will make the suggested change.

Reply:
done!